demos/blockfactory/cp.js

```diff
@@ -3,18 +3,7 @@
  *
  * Copyright 2006 Neil Fraser
  * https://neil.fraser.name/software/colourpicker/
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
I am Neil Fraser, and I approve the change to this file.
(Google is not allowed to convert licenses to SPDX in non-Google files.)